image-viewer.py

```diff
@@ -56,72 +59,91 @@ def render(self):
 
         self.root.bind("<KeyPress>", self.onKeyPress)
         self.root.mainloop()
-        
+
     def create_menu(self):
         menubar = Menu(self.root)
         self.root.config(menu=menubar)
-        
+
         file_menu = Menu(menubar, tearoff=False)
-        file_menu.add_command(label='Open Folder', command=self.open_folder, underline=0)
-        file_menu.add_command(label='Refresh', command=self.refresh, underline=0)
-        file_menu.add_command(label='Exit', command=self.root.destroy, underline=1)
-        menubar.add_cascade(label='File', menu=file_menu, underline=0)
-    
+        file_menu.add_command(label="Open Folder", command=self.open_folder, underline=0)
+        file_menu.add_command(label="Refresh", command=self.refresh, underline=0)
+        file_menu.add_command(label="Exit", command=self.root.destroy, underline=1)
+
+        theme_menu = Menu(menubar, tearoff=0)
+        theme_menu.add_command(label="Dark", command=self.set_theme_dark, underline=0)
+        theme_menu.add_command(label="Light", command=self.set_theme_light, underline=0)
+        file_menu.add_cascade(label="Theme", menu=theme_menu, underline=0)
```

Comment:
Can "Theme" come under a separate menu please called "Customise" because I think that would be a more obvious place for a user, and allow opportunities for more ways to customise the image viewer in the future.
Customise should be to the right of "File"

Reply:
Yes, For Sure i will do it now